express/code/blocks/ax-columns/ax-columns.css

```diff
@@ -33,6 +33,24 @@ main .section .ax-columns.enterprise .column h2 {
   font-size: 60px;
 }
 
+main .content.columns-medium-header h1,
```

Comment:
in general we should avoid putting a main here, unless it's specifically intentional that we won't want to use these styles in a fragment/modal. To make rules more specific we should use other structures like block names